waltz-tools/src/main/java/com/wepay/waltz/tools/client/ClientCli.java

```diff
@@ -44,18 +44,15 @@
     private ClientCli(String[] args,  boolean useByTest) {
         super(args, useByTest, Arrays.asList(
                 new Subcommand(Validate.NAME, Validate.DESCRIPTION, Validate::new),
-                new Subcommand(HighWaterMark.NAME, HighWaterMark.DESCRIPTION, HighWaterMark::new)
-        ));
+                new Subcommand(HighWaterMark.NAME, HighWaterMark.DESCRIPTION, HighWaterMark::new),
+                new Subcommand(Producer.NAME, Producer.DESCRIPTION, Producer::new),
+                new Subcommand(Consumer.NAME, Consumer.DESCRIPTION, Consumer::new)
+                ));
     }
 
     /**
-     * Use {@code Validate} command to submit transactions to Waltz
-     * server for testing, and consume the transactions for validation,
-     * including transaction data and optimistic locking.
-     *
-     * When running {@code Validate} command, there should not be any external
-     * client that is producing TXNs of the same partition (different partition
-     * is okay). Otherwise, the validation result will be unpredictable.
+     * {@code ClientCliBaseClass} serves as a parent class other client classes build on top of.
```

Comment:
nit: `{@code ClientCliBaseClass} serves as a base class for few Client Cli classes.`